coop_membership/views/web_templates.xml

```diff
@@ -0,0 +1,569 @@
+                            </div>
+                            <!-- <div class="wrap-input100 bg1 input_radio" >
+                                <span class="label-input100 long-text">Titulaire d'un justificatif minima social</span>
+                                <br />
+                                <input  type="radio" name="social_registration" value="yes">  Oui </input><br />
+                                <input  type="radio" checked="checked" name="social_registration" value="no">  Non </input><br />
+                            </div> -->
+
+                            <div class="wrap-input100 input_radio validate-input bg1 dropDownSelect2">
+                                <span class="label-input100 long-text">Je m'inscris à la réunion d'accueil du :</span>
+                                <br/>
+                                <span style="color: #f04a4f" class="validate-option-event">Veuillez renseigner votre
+                                    réunion
+                                </span>
+                                <div>
+                                    <t t-foreach="datas" t-as="data">
+                                        <input type="radio" name="select_event"
+                                               class="validate-input-option validate-input" t-att-value="data[0]">
+                                            <t t-esc="data[2][0:10]"/>
+                                            à
+                                            <t t-esc="data[2][11:16] or False"/>
+                                            <t t-if="data[1]">-
+                                                <t t-esc="data[1]"/>
+                                            </t>
+                                        </input>
+                                        <br/>
+                                    </t>
+                                </div>
+                            </div>
+
+                            <div class="g-recaptcha" t-att-data-sitekey="captcha_site_key"
+                                 data-validate="Veuillez cliquer sur la case reCAPTCHA"/>
+                            <div style="color: #f04a4f" class="validate-option-event">Veuillez cliquer sur la case
+                                reCAPTCHA
+                            </div>
+
+                            <div class="container-contact100-form-btn">
+                                <button class="contact100-form-btn">
+                                    <span>
+                                        Envoyer
+                                        <i class="fa fa-long-arrow-right m-l-7" aria-hidden="true"></i>
+                                    </span>
+                                </button>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+
+                <script type="text/javascript" src="/coop_membership/static/lib/jquery/jquery-3.2.1.min.js"></script>
```

Comment:
This is weird. Do we really need to include jQuery library?
Maybe in 9.0 there was an old version of jQuery, but in 12.0 we no longer need to include it?

Can you analize and see if it can be removed?

Reply:
@ivantodorovich : In odoo 12 still used old version of jquery (1.11.1) : https://github.com/odoo/odoo/blob/12.0/addons/web/static/lib/jquery/jquery.js#L2

If we remove it, some feature will not work like: date picker in Date of Birth field.